### DISCUSSION TITLE
Netlify build fail with: Query engine library for current platform "debian-openssl-1.1.x" could not be found.

### DISCUSSION THREAD

**Discussion by aaron-henselin** [prisma/prisma#14568] [Q&A]
When a **cached** Netlify build occurs, the query engine library cannot be found, and it advises to create a ticket.
Clearing cache and retry resolves the issue until the next build.

Can you advise on what needs to change (either in configuration, in the Netlify platform, or other). 

Build log:

> 10:22:03 PM: > Build error occurred
10:22:03 PM: Error: Query engine library for current platform "debian-openssl-1.1.x" could not be found.
10:22:03 PM: You incorrectly pinned it to debian-openssl-1.1.x
10:22:03 PM: This probably happens, because you built Prisma Client on a different platform.
10:22:03 PM: (Prisma Client looked in "/opt/build/repo/node_modules/@prisma/client/runtime/libquery_engine-debian-openssl-1.1.x.so.node")
10:22:03 PM: Searched Locations:
10:22:03 PM:   /opt/build/repo/node_modules/.prisma/client
10:22:03 PM:   /opt/build/repo/node_modules/@prisma/client
10:22:03 PM:   /opt/build/repo/node_modules/@prisma/client
10:22:03 PM:   /opt/build/repo/node_modules/.prisma/client
10:22:03 PM:   /opt/build/repo/prisma
10:22:03 PM:   /tmp/prisma-engines
10:22:03 PM:   /opt/build/repo/node_modules/.prisma/client
10:22:03 PM: You already added the platforms "debian-openssl-1.1.x", "debian-openssl-1.1.x" to the "generator" block
10:22:03 PM: in the "schema.prisma" file as described in https://pris.ly/d/client-generator,
10:22:03 PM: but something went wrong. That's suboptimal.
10:22:03 PM: Please create an issue at https://github.com/prisma/prisma/issues/new
10:22:03 PM:     at RequestHandler.handleRequestError (/opt/build/repo/node_modules/@prisma/client/runtime/index.js:28664:13)
10:22:03 PM:     at PrismaClient._getDmmf (/opt/build/repo/node_modules/@prisma/client/runtime/index.js:29740:23)
10:22:03 PM:     at async PrismaClient._executeRequest (/opt/build/repo/node_modules/@prisma/client/runtime/index.js:29661:22)
10:22:03 PM:     at async consumer (/opt/build/repo/node_modules/@prisma/client/runtime/index.js:29618:18)
10:22:03 PM:     at async PrismaClient._request (/opt/build/repo/node_modules/@prisma/client/runtime/index.js:29639:16)
**** exluded for brevity ***
10:22:03 PM:     at async buildStaticPaths (/opt/build/repo/node_modules/next/dist/build/utils.js:487:31)
10:22:03 PM:     at async /opt/build/repo/node_modules/next/dist/build/utils.js:622:115
10:22:03 PM:     at async Span.traceAsyncFn (/opt/build/repo/node_modules/next/dist/trace/trace.js:79:20) {
10:22:03 PM:   type: 'PrismaClientInitializationError',
10:22:03 PM:   clientVersion: '4.1.0'

**Comment by ludralph** [comment]
Hi @aaron-henselin 👋 

Are you specifying the generator block in your schema file like the snippet below

```js
generator prisma {
  provider = "prisma-client-js"
  binaryTargets = ["native","darwin","debian-openssl-1.1.x"]
}
```
You can also take a look at a related issue [here](https://github.com/prisma/prisma-client-js/issues/616) and this Netlify [link](https://answers.netlify.com/t/support-guide-my-site-deploy-fails-unless-netlifys-build-cache-is-cleared/3035).

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
